DQMOffline/Muon/python/diMuonHistograms_cfi.py

```diff
@@ -25,3 +25,15 @@
                               HighMassMin = cms.double(55.0),
                               HighMassMax = cms.double(155.0)
                               )
+
+from Configuration.StandardSequences.Eras import eras
+eras.pA_2016.toModify(diMuonHistos,
+                      etaBin = 350,
+                      etaBBin = 350,
+                      etaEBin = 350,
+
```

Comment:
Nevermind, after checking the code more carefully I see that if the etaBinLM is zero, it uses the same binning as the Z mass plot, which is what is wanted in thus case. Then everything is fine and you can ignore my comment